config/express.js

```diff
@@ -49,7 +50,24 @@ app.use(helmet());
 // enable CORS - Cross Origin Resource Sharing
 app.use(cors());
 
-app.use(session({ secret: config.cookieSecret, resave: true, saveUninitialized: true }));
+/* const store = new MongoDBStore({
+  mongooseConnection: mongoose.connection,
+  uri: 'mongodb://localhost/session_hackathon',
```

Comment:
Not sure if you need the uri if you have a mongooseConnection

Reply:
This stores the session in a separate collection from the main one.

Reply:
...err, separate database I think